documentation/openapi-authoring-automated-guidelines.md

```diff
@@ -143,7 +150,8 @@ The date MAY optionally be followed by one of:
 
 Links: [Index](#index) | [Error vs. Warning](#error-vs-warning) | [Automated Rules](#automated-rules) | [RPC](#rpc-violations): [Errors](#rpc-errors) or [Warnings](#rpc-warnings) | [SDK](#sdk-violations): [Errors](#sdk-errors) or [Warnings](#sdk-warnings)
 
-### <a name="M3016" />M3016 PropertiesNamesCamelCase
+### <a name="R3014" />R3014 BodyPropertiesNamesCamelCase
+### <a name="R3016" />R3016 DefinitionsPropertiesNamesCamelCase
```

Comment:
Agreed but I doubt if we update documentation that often, I will leave it to your discretion.